custom/ewsghana/tests/handlers/stock_on_hand.py

```diff
@@ -0,0 +1,115 @@
+from custom.ewsghana.tests.handlers.utils import EWSScriptTest
+
+
+class StockOnHandTest(EWSScriptTest):
+
+    def test_stock_on_hand(self):
+        a = """
+           5551234 > soh lf 31.0
+           5551234 < Dear stella, thank you for reporting the commodities you have in stock.
+           5551234 > soh lf 31.0 mc 25.0
+           5551234 < Dear stella, thank you for reporting the commodities you have in stock.
+           5551234 > SOH LF 31.0 MC 25.0
+           5551234 < Dear stella, thank you for reporting the commodities you have in stock.
```

Comment:
actually @rowenaluk can you take a look through these? Imagine since they were ported over from v1 that you're the best person to vet them.